airflow/providers/google/marketing_platform/hooks/display_video.py

```diff
@@ -151,3 +152,21 @@ def upload_line_items(self, line_items: Any) -> List[Dict[str, Any]]:
             .execute(num_retries=self.num_retries)
         )
         return response
+
+    def download_line_items(self, request_body: Dict[str, Any]) -> List[Any]:
+        """
+        Retrieves line items in CSV format.
```

Comment:
```suggestion
        Retrieves line items.
```
`List[Any]` is not a CSV format I think

Reply:
Documentation points to CSV.
https://developers.google.com/bid-manager/v1.1/lineitems/downloadlineitems